### PR TITLE
[ch28795] Rename all references to old name

### PR DESCRIPTION
**Pull request by victorpoluceno**
Remove all references to the old cf-review-name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,9 +20,6 @@ When the new `chart version` is created the CI pipeline run and push the new cha
 
 | Variable  | Value |
 |----- |-------|
-| KUBE_CONTEXT | k8s-context |
```
Comment (victorpoluceno) [inline README.md:23]: For testing purposes we only need to set the CHART_VERSION. I will later work on a proper docs that will explain how to run it.

File: prune/main.go [hunk 2]
```diff
@@ -254,7 +254,7 @@ func (k8s *K8sClient) DeleteNamespace(name string) error {
 func (k8s *K8sClient) NamespaceList() ([]K8sNamespace, error) {
 	namespacesClient := k8s.ClientSet.CoreV1().Namespaces()
 	namespaceList, err := namespacesClient.List(metav1.ListOptions{
-		LabelSelector: "app.kubernetes.io/name=cf-review-env",
+		LabelSelector: "app.kubernetes.io/name in (cf-review-env, kube-review)",
```
Comment (victorpoluceno) [inline prune/main.go:257]: This is keep reviews running for a few days with the old label, after that we can remove this.

**Review by rodrigo-findhotel — APPROVED** [review]
Good luck! 🥳

File: Dockerfile [hunk 3]
```diff
@@ -16,14 +16,10 @@ RUN chmod +x prune
 FROM codefresh/cfstep-helm:3.1.1
 
 ARG DEFAULT_HELM_REPO_URL
-ARG KUBE_CONTEXT
```
Comment (victorpoluceno) [inline Dockerfile:19]: I'm also taking the opportunity to move this to the client. It makes the kube review easier to setup, as it's up to each project on what cluster and what domain they will use to deploy.

**Pull request merged**